setup.py

```diff
@@ -43,6 +43,11 @@ def parse_requires():
 with open("README.md") as f:
     long_desc = f.read()
 
+
+extensions = [
+    Extension('uarray._uarray', sources=['uarray/_uarray_dispatch.cxx'], language='c++11')
```

Comment:
Interesting...

```python
```suggestion
    Extension('uarray._uarray', sources=['uarray/_uarray_dispatch.cxx'], language='c++11', extra_compile_args=['-std=c++11'])
```

```